tests/unit/test_charm_leader_unit.py

```diff
@@ -67,18 +67,6 @@ def test_given_alertmanager_relation_not_created_when_pebble_ready_then_charm_go
             "Waiting for alertmanager relation to be created"
         )
 
-    @patch("charm.AlertmanagerConfigDirWatcher", Mock())
```

Comment:
Why the removal of all those tests?

Reply:
New `ops.testing`, on `container_pebble_ready` automatically sets `can_connect` to True, which makes this test useless.